root/entity/Aliases.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import chooseLayoutComponent from '../utility/chooseLayoutComponent';
+import AliasesComponent from '../components/Aliases';
```

Comment:
`AliasesComponent` doesn’t exist

Reply:
AliasesComponents is an alias for the default export of components/Aliases because this one is also Aliases.

Reply:
Well, the only other way is to do `import {default as AliasesComponent}` but I'm not sure that's an improvement.